stack.yaml

```diff
@@ -0,0 +1,32 @@
+resolver: lts-14.9
```

Comment:
I don't understand why you need this at the top level AND in the subdirectories.  Isn't that yet another duplication?  Can we just do it at the top level?  That would be far more tolerable.

Reply:
Yes